_notices/rgn0024.md

```diff
@@ -27,18 +27,18 @@ notice_updated: 2022-04-27
 ## Overview
 
 The installation method for a RAPIDS compatible WSL2 instance has changed with the release of NVIDIA's WSL2 
-capable Production Drivers.  If you already have RAPIDS running on WSL2, Please 
-[update your drivers](https://www.nvidia.com/Download/index.aspx?lang=en-us) 
+capable Windows 11 Production Drivers.  If you already have RAPIDS running on WSL2, Please 
+[update your Windows drivers](https://www.nvidia.com/Download/index.aspx?lang=en-us) 
 accordingly for continued EA support with RAPIDS `v22.02`.  If you are installing RAPIDS inside WSL2 for the 
 first time, please follow the updated instructions below:
 
 	1. Please follow the steps in this MS Blog to install WSL2: https://docs.microsoft.com/en-us/windows/wsl/install
-	2. Install the latest production drivers, which are WSL2 capable: https://www.nvidia.com/Download/index.aspx?lang=en-us
+	2. Install the latest Windows 11 production drivers on the host machine, which are WSL2 capable: https://www.nvidia.com/Download/index.aspx?lang=en-us
 	3. Inside WSL2:
 		a. Install conda or docker environments
 		b. Install RAPIDS as normal with your environment options of choice.
 
-Please do not install any additional NVIDIA drivers inside WSL2.  It is no longer necessary.
+This makes a complete RAPIDS installation far easier in WSL2.  Please do not install any additional NVIDIA drivers, neither GPU nor CUDA, inside your WSL2 instance as it is no longer necessary.  CUDA Toolkit (CTK) will be installed when you install RAPIDS in your desired environment.  
```

Comment:
Nitpick, non blocking: `This makes a complete RAPIDS installation far easier in WSL2.` isn't necessary, and it's not totally clear to me what "this" is.